docs/markdown/Writing Plugins/common-plugin-tasks/plugin-upgrade-guide.md

```diff
@@ -126,7 +126,7 @@ The partitioner rule gives you all the matching files (or `FieldSet`s depending
 
```

Comment:
I think was done as part of a Find+Replace? I don't think we should be changing the plugin upgrade guide since `FmtRequest` was the name of the type plugin authors would've used prior to the document changes.

Reply:
I actually found this being somewhere in the middle ground. Sure I can remove it again